tests/mixed-source/ios/MixedSourceFramework/Logger/SwiftLogger.swift

```diff
@@ -0,0 +1,10 @@
+import Foundation
+
+@objc
+public class SwiftLogger: NSObject {
```

Comment:
@segiddins So I am running into an issue locally where classes that are exposed to ObjC must be `public` in order to work. I have verified this by changing this class to internal and it does not work. Is this expected behavior?

Reply:
are you passing `-extension` via xcode, by any chance? that changes swift to emit `internal` symbols into the generated objc header

Reply:
Do you mean `-application-extension` or `-extension`?

Reply:
Unsure off the top of my head

Reply:
Do you know where this flag is documented? I would like to read more about it. Thanks

Reply:
https://github.com/apple/swift/blob/86a8d577ed5c8863def050239fb557f9ebea1904/lib/Frontend/CompilerInvocation.cpp#L1899-L1924 from `-fapplication-extension`

Reply:
This led me down a path of just fighting with `application-extension`, I am just going to take my medicine and make the types `public`